packages/controls/src/version.ts

```diff
@@ -6,4 +6,4 @@
  * implements.
  */
 export
-const JUPYTER_CONTROLS_VERSION = '1.4.0';
+const JUPYTER_CONTROLS_VERSION = require('../package.json').version;
```

Comment:
@vidartf I am reading that value from `package.json`. It appears I cannot use the `import` syntax because of the ts root dir though.

Reply:
No, this is not the package.json version number. It's the controls *spec* version number, which may be different from the package.json version number. For example, you'll notice that these are already different. package.json has version 1.4.5, while this is 1.4.0. This allows us to bump the package version without bumping the attribute spec version.

Reply:
Our guarantee is that a bump in the spec version will bump the package version, but if something changes in the package that does not change the spec, the package version can bump without changing the spec version.

Reply:
Did the spec change in a backwards incompatible way? If not, then the controls spec should go to 1.5, while the package version should go to 2.0

Reply:
(that would allow this lab manager to work with ipywidgets 7.4, which asks for controls *spec* `^1.4`...)

Reply:
@jasongrout Doesn't this break the managers that use the version string for CDN?

Reply:
@vidartf this is an excellent point.